.vuepress/public/_redirects

```diff
@@ -203,3 +203,10 @@
 # New "Update and migration" section
 /learn/getting_started/algolia_migration.html       /learn/update_and_migration/algolia_migration.html
 /learn/advanced/updating.html                       /learn/update_and_migration/updating.html
+
+# Move faceting to a new guide
+/learn/advanced/filtering_and_faceted_search.html   /learn/advanced/faceted_search.html
+
+# Rename filtering guide
+/learn/advanced/filtering_and_faceted_search.html   /learn/advanced/filtering_guide.html
```

Comment:
How is this going to be resolved since there are two destinations?

Reply:
Sorry, I didn't get what you mean 🙏

Reply:
I get it! Updated the link